cli/cli.hs

```diff
@@ -207,6 +210,7 @@ main = withUtf8 $ do
     , abstRefineMem = cmd.abstractMemory
     , abstRefineArith = cmd.abstractArithmetic
     , dumpTrace = cmd.trace
+    , decomposeStorage = Prelude.not cmd.nodecompose
```

Comment:
`cmd.noDecompose`

Reply:
Hehe, yes, sorry. I fixed it, now it's passing :)